lib/db.js

```diff
@@ -1107,6 +1107,45 @@ module.exports = (
     return this.pool.post(`/account/${uid}/resetTokens`)
   }
 
+  DB.prototype.createTotpToken = function (uid, sharedSecret, epoch) {
```

Comment:
It's a nit, but I wonder if there's a better name for these than "TOTP Tokens" since we have so many other things in this repo called "XYZToken".  But maybe calling them "tokens" is standard in other implementations?

Reply:
Fwiw, I wondered the same thing when reviewing the db PR so I went to look at the spec text. They do use "token" there, so it seems like a sensible choice in that sense...

Reply:
You had me at "spec" 👍

Reply:
I considered calling them devices or configuration, but opted for tokens because it seemed more sensible. Good to know that the spec referred to them as tokens as well.